src/css/proposal-to-leave/proposal-to-leave-desktop.css

```diff
@@ -0,0 +1,79 @@
+/* Desktop styles */
+@media (min-width: 1158px) {
+  .container.proposal-to-leave {
+    width: calc(1152px - 20px);
+    padding: 60px;
```

Comment:
don't review padding and width for .container, that's the way how you missed correct width and paddings for mobile

Reply:
also we have separate task for add spaces between sections